docs/docs/building-a-contact-form.md

```diff
@@ -43,6 +43,32 @@ Each method detailed below will start with the following contact form:
 
 ## Form submission options in Gatsby
 
+### Getform
+
+Getform is a form backend platform which offers a free-plan for handling form submissions on static sites. In order to begin receiving form submissions from your Gatsby site, you must first create a form on the Gatsby site. When creating the form, direct the http POST method to the Getform, by placing the `name` attributes for the fields you want to make visible. (name, email, message etc.)
+
+```jsx:title=src/pages/contact.js
+<form method="post" action="https://getform.io/{your-unique-getform-endpoint}">
+  ...
+  <label>
+    Email
+    <input type="email" name="email" />
+  </label>
+  <label>
+    Name
+    <input type="text" name="name" />
+  </label>
+  <label>
+    Message
+    <input type="text" name="message" />
+  </label>
+  ...
+</form>
+```
+Once you've made the changes on your Gatsby contact form, you can head over to your contact page and start receiving submissions to the form, the submissions will then be visible on Getform dashboard. You can add multiple email addresses to receive email notifications for the forms created, as well as manipulate the data you see on Getform using Zapier and Webhooks options that are offered. 
```

Comment:
```suggestion
Once you've made the code changes in your contact form, you can head over to the contact page on your site and start submitting data to the form. The submissions will then be visible on the Getform dashboard. You can add multiple email addresses to receive email notifications for the forms created, as well as manipulate the data you see on Getform using Zapier and Webhooks options that are offered. 
```
There are a few different levels here. The form code in Gatsby, the form itself in your site, and the Getform dashboard. I made some recommendations that I hope will clarify which piece we're referring to.